src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
```

Comment:
make UserService class-level variable to avoid creating object each time you call `login` method
```suggestion
    private final UserService userService = new UserService();
    /**
```